financial_time_series/README.md

```diff
@@ -0,0 +1,285 @@
+Using Kubeflow for Financial Time Series
+====================
+
+This repository is linked to a series of blogposts.
+The first blogpost can be found on: https://blog.ml6.eu/using-kubeflow-for-financial-time-series-18580ef5df0b
```

Comment:
Link instead of raw URL (or maybe this renders fine idk).

Reply:
Noted - I removed the link to the blogpost as it's not relevant in this context.